### PR TITLE
feat: Add article on Automated Accessibility Tools

### PR DESCRIPTION
**Pull request by rkrupnick**
- [ x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ x] My pull request targets the `master` branch of freeCodeCamp.
- [x ] None of my changes are plagiarized from another source without proper attribution.
- [x ] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x ] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

**Review by theNaut1lus — COMMENTED** [review]
Also add a short tutorial describing how to set up testing tools onto your repository or project.

the most common you can pick up is Lighthouse or something similar.

File: guide/english/accessibility/automated-testing/index.md [hunk 1]
```diff
@@ -3,11 +3,16 @@ title: Automated Accessibility Testing Tools
...
+They are testing to see if your content follows established guidelines such as Section 508 Compliance, Web Content Accessibility Guidelines (WCAG), and the Americans with Disabilities Act (ADA).
+
+### How Do They Work?
+This depends on the tool you are using, and the language or framework that your codebase is written in. Many integrate with a testing suite.
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
 
 #### More Information:
-<!-- Please add any articles you think might be helpful to read before writing the article -->
+[W3C List of Web Accessibility Tools](https://www.w3.org/WAI/ER/tools/)
```
Comment (moT01) [inline guide/english/accessibility/automated-testing/index.md]:
```suggestion
[W3C List of Web Accessibility Tools](https://www.w3.org/WAI/ER/tools/)  
```

File: guide/english/accessibility/automated-testing/index.md [hunk 2]
```diff
@@ -3,11 +3,16 @@ title: Automated Accessibility Testing Tools
...
+[W3C Guide to Selecting Accessibility Tools](https://www.w3.org/WAI/test-evaluate/tools/selecting/)
```
Comment (moT01) [inline guide/english/accessibility/automated-testing/index.md]:
```suggestion
[W3C Guide to Selecting Accessibility Tools](https://www.w3.org/WAI/test-evaluate/tools/selecting/)  
```

File: guide/english/accessibility/automated-testing/index.md [hunk 3]
```diff
@@ -3,11 +3,16 @@ title: Automated Accessibility Testing Tools
...
+[Comparing 3 Top Automated Accessibility Tools](https://medium.com/myplanet-musings/comparing-3-top-automated-accessibility-testing-tools-wave-tenon-io-and-google-lighthouse-d3897d7bb311)
```
Comment (moT01) [inline guide/english/accessibility/automated-testing/index.md]:
```suggestion
[Comparing 3 Top Automated Accessibility Tools](https://medium.com/myplanet-musings/comparing-3-top-automated-accessibility-testing-tools-wave-tenon-io-and-google-lighthouse-d3897d7bb311)  
```

**Review by moT01 — CHANGES_REQUESTED** [review]
Thanks for the PR @rkrupnick, it looks good - [here's a preview of your changes](https://github.com/freeCodeCamp/freeCodeCamp/blob/12941d7ed1db63309c55cdc5bfb4bd20822b17de/guide/english/accessibility/automated-testing/index.md) - the `More Information` area needs some line breaks. I believe adding two spaces at the end of each line will make that line break. It would be great if you could add those. Thanks.

**Review by moT01 — APPROVED** [review]
Looks good @rkrupnick - thanks for contributing to freeCodeCamp! 🎉

**Pull request merged**